pokemongo_bot/cell_workers/incubate_eggs.py

```diff
@@ -106,15 +106,13 @@ def _check_inventory(self, lookup_ids=[]):
                         "used": False
                     })
                 elif 'is_egg' not in pokemon and pokemon['id'] in lookup_ids:
-                    matched_pokemon.append({
-                        "pokemon_id": pokemon.get('pokemon_id', -1),
-                        "cp": pokemon.get('cp', 0),
+                    matched_pokemon.append(pokemon.update({
```

Comment:
It was initially a change required for an earlier iteration of this to work (as it required that pokemon objects passed in maintained their standard keys). As you can see, instead of passing out an object with only a subset of keys, it maintains the original object and just appends new calculated data. I opted to leave it in because I feel this is still a better approach in case the egg incubation worker wants to utilize other keys in the pokemon object. At the very least, it doesn't break anything.